service/src/main/resources/db/migration/R__message_account_name_view.sql

```diff
@@ -0,0 +1,17 @@
+CREATE VIEW message_account_name_view(
```

Comment:
I guess this could already start with 'drop view if exists' statement to make future changes easier

Reply:
Ach, of course. 👍